README.rst

```diff
@@ -255,6 +255,7 @@ Then Lightning-GPU with MPI support can then be installed in the *editable* mode
 
 .. code-block:: bash
 
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:`/path/to/libmpi.so`
```

Comment:
Let's add this incomplete instruction to that rst note instead. Users usually copy and paste these instructions without noticing `/path/to/libmpi.so`.